docs/integrations/sources/hubspot.md

```diff
@@ -100,30 +136,35 @@ The HubSpot source connector supports the following streams:
 
 First you need to give the connector some additional permissions:
 
 - **If you are using OAuth on Cloud** go to the Hubspot source settings page in the Airbyte UI and reauthenticate via Oauth to allow Airbyte the permissions to access custom objects.
-- **If you are using OAuth on OSS or Private App auth (on OSS or Cloud)**: you'll need to go into the Hubspot UI where you created your private app or oauth application and add the `crm.objects.custom.read` scope to your app's scopes. See Hubspot's instructions here.
+- **If you are using OAuth on OSS or Private App auth (on OSS or Cloud)**: you'll need to go into the Hubspot UI where you created your Private App or OAuth application and add the `crm.objects.custom.read` scope to your app's scopes. See HubSpot's instructions [here](https://developers.hubspot.com/docs/api/working-with-oauth#scopes).
 
-Then, go to the replication settings of your connection and click “refresh source schema” to pull in those new streams for syncing.
+Then, go to the replication settings of your connection and click **refresh source schema** to pull in those new streams for syncing.
 
-### A note on the `engagements` stream
-
-Objects in the `engagements` stream can have one of the following types: `note`, `email`, `task`, `meeting`, `call`. Depending on the type of engagement, different properties is set for that object in the `engagements_metadata` table in the destination:
+### Notes on the `engagements` stream
 
+1. Objects in the `engagements` stream can have one of the following types: `note`, `email`, `task`, `meeting`, `call`. Depending on the type of engagement, different properties are set for that object in the `engagements_metadata` table in the destination:
 - A `call` engagement has a corresponding `engagements_metadata` object with non-null values in the `toNumber`, `fromNumber`, `status`, `externalId`, `durationMilliseconds`, `externalAccountId`, `recordingUrl`, `body`, and `disposition` columns.
 - An `email` engagement has a corresponding `engagements_metadata` object with non-null values in the `subject`, `html`, and `text` columns. In addition, there will be records in four related tables, `engagements_metadata_from`, `engagements_metadata_to`, `engagements_metadata_cc`, `engagements_metadata_bcc`.
 - A `meeting` engagement has a corresponding `engagements_metadata` object with non-null values in the `body`, `startTime`, `endTime`, and `title` columns.
 - A `note` engagement has a corresponding `engagements_metadata` object with non-null values in the `body` column.
 - A `task` engagement has a corresponding `engagements_metadata` object with non-null values in the `body`, `status`, and `forObjectType` columns.
 
+2. The `engagements` stream uses two types of API:
+- **EngagementsRecent** if `start_date/last_sync_date` is less than 30 days and the API is able to return all records (<10k).
```

Comment:
The wording here is a bit rough, we should rephrase to make it clearer. Fundamentally this section is saying: "FYI this stream might take a while to sync if it's been a while since you synced data or if there's a ton of data generated since the last sync, so we recommend scheduling frequent syncs"

Reply:
Changed this to hopefully be clearer. Now reads:

```
2. The `engagements` stream uses two different APIs based on the length of time since the last sync and the number of records.
- **EngagementsRecent** if the following two criteria are met:
    - The last sync was performed within the last 30 days
    - Fewer than 10,000 records are being synced
- **EngagementsAll** if either of these criteria are not met.

Because of this, the `engagements` stream can be slow to sync if it hasn't synced within the last 30 days and/or is generating large volumes of new data. We therefore recommend scheduling frequent syncs.
```

Let me know if it's still not quite there yet!